crates/router/src/core/payments/operations/payment_update.rs

```diff
@@ -919,6 +920,11 @@ impl<F: Clone> UpdateTracker<F, PaymentData<F>, api::PaymentsRequest> for Paymen
             )
             .await
             .to_not_found_response(errors::ApiErrorResponse::PaymentNotFound)?;
+        req_state
+            .event_context
+            .event(AuditEvent::new(AuditEventType::PaymentUpdate))
+            .with(payment_data.to_event())
+            .emit();
```

Comment:
Can you send `amount` to be logged here
You can take reference from this PR: https://github.com/juspay/hyperswitch/pull/4913/files

Reply:
I have added the amount in the fresh commit.